src/EditorFeatures/CSharpTest/CodeActions/ReplacePropertyWithMethods/ReplacePropertyWithMethodsTests.cs

```diff
@@ -1325,6 +1325,97 @@ private int GetProp()
 }", ignoreTrivia: false, options: PreferExpressionBodiedMethods);
         }
 
+        [WorkItem(18234, "https://github.com/dotnet/roslyn/issues/18234")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplacePropertyWithMethods)]
+        public async Task TestDocumentationComment1()
+        {
+            await TestInRegularAndScriptAsync(
+@"internal interface ILanguageServiceHost
+{
+    /// <summary>
+    ///     Gets the active workspace project context that provides access to the language service for the active configured project.
+    /// </summary>
+    /// <value>
+    ///     An value that provides access to the language service for the active configured project.
```

Comment:
I often find test method names pointless.  I want the test content to simply be self descriptive.  If i could make most tests anonymous, i would. :)